pages/common/nix-shell.md

```diff
@@ -0,0 +1,28 @@
+# nix-shell
+
+> Start an interactive shell based on a Nix expression.
+> More information: <https://nixos.org/manual/nix/stable/#sec-nix-shell>.
+
+- Start with nix expression in ./shell.nix or ./default.nix:
+
+`nix-shell`
+
+- Run shell command in non-interactive shell and exit:
+
+`nix-shell --run {{cmd}}`
+
+- Start with expression in default.nix in the current directory:
+
+`nix-shell default.nix`
+
+- Start with packages loaded from nixpkgs:
+
+`nix-shell -p {{pkg_name_1}} {{pkg_name_2}}`
+
+- Start with packages loaded from specific nixpkgs revision:
+
+`nix-shell -p {{pkg_names}} -I nixpkgs=https://github.com/NixOS/nixpkgs-channels/archive/{{nixpkgs_revision_hash}}.tar.gz`
```

Comment:
I'm not sure about this URL.
```suggestion
`nix-shell --packages {{package_names}} -I nixpkgs={{https://github.com/NixOS/nixpkgs-channels/archive/{{nixpkgs_revision_hash}}.tar.gz}}`
```

Reply:
Looks like this is a deprecated URL, thank you! I pulled it out of the official nix-manual, but it looks like it may need to be updated there too. 

Your suggested change has a token in a token, is that allowed?